### PR TITLE
workaround for localized strings

### PR DESCRIPTION
**Pull request by ferbar**
add: README.md

fix: set LANG=C before svn info | grep something

### CODE REVIEW

File: plugins/svn/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## svn
+
+**Maintainer:** 
+
+This plugin adds some useful functions.
+
+### Usage
+
+See the [wiki](https://github.com/robbyrussell/oh-my-zsh/wiki/Plugin:svn) for a list of functions provided by the plugin.
```
Comment (mcornella) [inline plugins/svn/README.md]: I'd rather have them listed here. We're trying to stop using the wiki for plugin documentation.

File: plugins/svn/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## svn
+
+**Maintainer:**
```
Comment (mcornella) [inline plugins/svn/README.md]: This is not needed.

File: plugins/svn/README.md (hunk 1 above)
Comment (mcornella) [inline plugins/svn/README.md]: I'd rather have them listed here. We're trying to stop using the wiki for plugin documentation.

File: plugins/svn/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## svn
+
+**Maintainer:** 
+
+This plugin adds some useful functions.
```
Comment (mcornella) [inline plugins/svn/README.md]: Expand the description (`useful functions for regular svn users`).
Also, add instructions on how to enable the plugin. See [example](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/gb/README.md).

File: plugins/svn/svn.plugin.zsh [hunk 4]
```diff
@@ -25,14 +25,14 @@ function in_svn() {
 
 function svn_get_repo_name() {
   if in_svn; then
-    svn info | sed -n 's/Repository\ Root:\ .*\///p' | read SVN_ROOT
-    svn info | sed -n "s/URL:\ .*$SVN_ROOT\///p"
+    LANG=C svn info | sed -n 's/Repository\ Root:\ .*\///p' | read SVN_ROOT
```
Comment (mcornella) [inline plugins/svn/svn.plugin.zsh]: I'd add the caret here as well: `^Repository`.

File: plugins/svn/README.md (hunk 1 above)
Comment (mcornella) [inline plugins/svn/README.md]: When you integrate the how-to use it with agnoster, add it after the `Functions` section, inside a `Usage` section.

**Comment by ferbar** [comment]
done

File: plugins/svn/README.md [hunk 5]
```diff
@@ -1,10 +1,62 @@
-## svn
+# `svn` plugin
 
-**Maintainer:** 
+This plugin adds some utility functions to display additional information regarding your current svn reositiory. See http://subversion.apache.org/ for the full svn documentation
```
Comment (mcornella) [inline plugins/svn/README.md]: - Typo on *repository*.
- Add a newline after *current*.
- End with a final dot.

File: plugins/svn/README.md [hunk 6]
```diff
@@ -1,10 +1,62 @@
...
-This plugin adds some useful functions.
+## Functions
 
-### Usage
+| Command                | Description                             |
+|:-----------------------|:----------------------------------------|
+|svn_prompt_info         | prompt for some themes                  |
+|in_svn                  | within svn directory                    |
+|svn_get_repo_name       |                                         |
+|svn_get_branch_name     | branch name (bug: returns the first path element if branches not used) |
```
Comment (mcornella) [inline plugins/svn/README.md]: Replace the bug part with `(see caveats)`.

File: plugins/svn/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+# `svn` plugin
+
+This plugin adds some utility functions to display additional information regarding your current
+svn repsitiory. See http://subversion.apache.org/ for the full svn documentation.
```
Comment (mcornella) [inline plugins/svn/README.md]: s/repsitiory/repository

**Pull request closed without merging**